### ISSUE TITLE
Sharp v0.33.5 not installing with Deno in github workflow runner

### ISSUE THREAD

**Issue by jasfour-seriesai** [lovell/sharp#4256]
## Possible install-time or require-time problem

- [x] I have read and understood all of the [documentation relating to installation](https://sharp.pixelplumbing.com/install).
- [x] I have searched for known bugs relating to this problem in my choice of package manager.

You must confirm both of these before continuing.

### Are you using the latest version of sharp?

- [x] I am using the latest version of `sharp` as reported by `npm view sharp dist-tags.latest`.

If you cannot confirm this, please upgrade to the latest version and try again before opening an issue.

If you are using another package which depends on a version of `sharp` that is not the latest,
please open an issue against that package instead.

### Are you using a supported runtime?

- [ ] I am using Node.js with a version that satisfies `^18.17.0 || ^20.3.0 || >=21.0.0`
- [x] I am using Deno
- [ ] I am using Bun

If you cannot confirm any of these,
please upgrade to the latest version
and try again before opening an issue.

### Are you using a supported package manager and installing optional dependencies?

- [ ] I am using npm >= 10.1.0 with `--include=optional`
- [ ] I am using yarn >= 3.2.0
- [ ] I am using pnpm >= 7.1.0 with `--no-optional=false`
- [x] I am using Deno
- [ ] I am using Bun

If you cannot confirm any of these, please upgrade to the latest version of your chosen package manager
and ensure you are allowing the installation of optional or multi-platform dependencies before opening an issue.

### What is the complete error message, including the full stack trace?

The trace from `deno install --allow-scripts=npm:sharp@0.33.5` running in a github runner is:

```
Initialize sharp@0.33.5: running 'install' script
error: script 'install' in 'sharp@0.33.5' failed with exit code 1
stderr:
Warning experimentalDecorators compiler option is deprecated and may be removed at any time
error: [ERR_MODULE_NOT_FOUND] Cannot find module 'file:///home/runner/work/mm-bot/mm-bot/node_modules/.deno/sharp@0.33.5/node_modules/sharp/install/check' imported from 'file:///home/runner/work/mm-bot/mm-bot/node_modules/.deno/sharp@0.33.5/node_modules/sharp'
error: failed to run scripts for packages: sharp@0.33.5
```

**Comment by lovell** [comment]
You probably don't need install-time scripts so should be able to remove the `--allow-scripts=npm:sharp@0.33.5` flag.

**Comment by lovell** [comment]
The install script is only needed for those using a globally-installed libvips and attempting to build sharp from source.

Adding the `--unstable-detect-cjs` flag will allow the install script to run, but it's effectively a no-op when no globally-installed libvips is found.

I wonder if there's a way to suppress this kind of warning in Deno? Other packages like esbuild will be similarly affected.

**Comment by lovell** [comment]
Thanks for the update, I'll close for now.

**Comment by pawsaw** [comment]
@lovell can we reopen this please?

```
% deno install --allow-scripts=npm:sharp@0.33.5
error: script 'install' in 'sharp@0.33.5' failed with exit code 1
stderr:
error: [ERR_MODULE_NOT_FOUND] Cannot find module 'file:///Users/pawel/devspark-web/node_modules/.deno/sharp@0.33.5/node_modules/sharp/install/check' imported from 'file:///Users/pawel/devspark-web/node_modules/.deno/sharp@0.33.5/node_modules/sharp'

error: failed to run scripts for packages: sharp@0.33.5
```

sharp is a dependency in nextjs 15. This is a major blocker while running nextjs on deno.

**Comment by lovell** [comment]
@pawsaw You don't need `--allow-scripts=npm:sharp@0.33.5`, please remove it.

**Comment by lovell** [comment]
That looks like an erroneous warning from Deno - see also https://github.com/denoland/deno/issues/26742

**Comment by pawsaw** [comment]
> That looks like an erroneous warning from Deno - see also [denoland/deno#26742](https://github.com/denoland/deno/issues/26742)

Thank you. Seems to be addressed in Deno 2.1.0
